src/components/dashboard/main-content/course-enrollments/course-cards/SavedForLaterCourseCard.jsx

```diff
@@ -0,0 +1,173 @@
+            course_run_id: courseRunId,
+          });
+        },
+        children: (
+          <div role="menuitem">
+            Move to In Progress
+            <span className="sr-only">for {title}</span>
+          </div>
+        ),
+      },
+    ]);
+  };
+
+  const renderButtons = () => {
+    if (isCourseEnded(endDate) || courseRunStatus === 'completed') { return null; }
+    return (
+      <ContinueLearningButton
+        linkToCourse={linkToCourse}
+        title={title}
+        courseRunId={courseRunId}
+      />
+    );
+  };
+
+  const renderCertificateInfo = () => (
+    props.linkToCertificate ? (
+      <div className="d-flex mb-3">
+        <div className="mr-3">
+          <img src={CertificateImg} alt="verified certificate preview" />
+        </div>
+        <div className="d-flex align-items-center">
+          <p className="lead mb-0 font-weight-normal">
+            View your certificate on{' '}
+            <a
+              className="text-underline"
+              href={`${process.env.LMS_BASE_URL}/u/${username}`}
+            >
+              your profile →
+            </a>
+          </p>
+        </div>
+      </div>
+    ) : (
+      <p className="lead mb-3 font-weight-normal">
+        To earn a certificate,{' '}
+        <a className="text-underline" href={props.linkToCourse}>
+          retake this course →
+        </a>
+      </p>
+    )
+  );
+
+  return (
+    <BaseCourseCard
+      buttons={renderButtons()}
+      dropdownMenuItems={getDropdownMenuItems()}
+      type="completed"
+      hasViewCertificateLink={false}
+      {...props}
+    >
+      {renderCertificateInfo()}
+      <MoveToInProgressModal
+        isOpen={isModalOpen}
+        courseTitle={title}
+        courseLink={linkToCourse}
+        courseId={courseRunId}
+        onClose={handleMoveToInProgressOnClose}
+        onSuccess={handleMoveToInProgressOnSuccess}
+      />
+    </BaseCourseCard>
+  );
+};
+
+SavedForLaterCourseCard.propTypes = {
+  linkToCourse: PropTypes.string.isRequired,
+  courseRunId: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  linkToCertificate: PropTypes.string,
+  savedForLater: PropTypes.bool.isRequired,
+  courseRunStatus: PropTypes.string.isRequired,
+  modifyCourseRunStatus: PropTypes.func.isRequired,
+  modifyIsMoveToInProgressCourseStatus: PropTypes.func.isRequired,
+  endDate: PropTypes.string,
+};
+
+SavedForLaterCourseCard.defaultProps = {
+  linkToCertificate: null,
+  endDate: null,
+};
+
+const mapDispatchToProps = dispatch => ({
+  modifyCourseRunStatus: (options) => {
+    // TODO remove this override of status once api is fixed or we find a better way to categorize the cards
+    // right now categorization is by status only, but it gets confusing if saved_for_later is false and
+    // status=complete for example
+    dispatch(updateCourseRunStatus({ ...options, status: 'in_progress' }));
```

Comment:
I had put in a hack to set to in_progress here.. due to how things were (ui only looked at course status before)... not sure is this is still needed, worth checking if removing still keeps everything working ...

Reply:
Initially this status: in_progress had to be here because the ui code only used course status as a way to separatee our the courses .I think with your latest changes, even if we remove this, things should work as expected ... I guess it would be good if we remove the status: 'in_progress' here and ensure everything still works and shows in the correct places ... is what I had meant, sorry was not too clear

basically I am overriding status coming from the backend which I think is not a good idea ...

Reply:
and if that works please remove that todo comment too. ..

Reply:
It worked!